client/src/components/ProductConfirmation.jsx

```diff
@@ -46,31 +90,24 @@ const ProductConfirmation = ({ setModal }) => {
       zIndex="10"
     >
       <Paper className={classes.paper}>
-        <Box
-          p={5}
-          display="flex"
-          flexDirection="column"
-          alignItems="center"
-          justifyContent="space-between"
-          height="90%"
+        <IconButton
+          className={classes.closeButton}
+          edge="start"
+          onClick={handleClose}
+          aria-label="close"
         >
-          <Typography variant="h6">
-            {title}
-            <Typography>Price: {price}</Typography>
-          </Typography>
-          <img className={classes.image} src={imageURL} alt={title}></img>
-          <Button
-            className={classes.button}
-            color="primary"
-            variant="contained"
-            onClick={handleClose}
-          >
-            Close
-          </Button>
-        </Box>
```

Comment:
As this part was removed from this branch, the add product from the dashboard would show a blank confirm page. Just make sure when merged to the dev this part of UI would be added back.

Reply:
yep. I put it back with `d3673a6`!